packages/terra-application-navigation/src/terra-dev-site/doc/example/ExampleApplication.jsx

```diff
@@ -141,6 +142,7 @@ const ExampleApplication = ({ onAction }) => {
   const [activeKey, setActiveKey] = useState(navigationItems[0].key);
 
   return (
+    <ExampleWrapper>
```

Comment:
Does this have lint errors for indentation?

Reply:
No

Reply:
It should, Please fix the indentation.

Reply:
Fixed 492f931